.github/workflows/kube-burner-configs/cluster-density-kube-burner.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
The location of these config files have to be moved, because CI is confusing these for github action files. I don't know where these should be put.

Reply:
I am not sure, maybe somewhere within `scripts` or `deploy` ?

Reply:
Since the config files need to be in test-gh-actions I decided to put them in scripts/release-tools/kube-burner-configs. They are not scripts, so they don't belong there, but maybe they could be considered to be release-tools.

Reply:
`scripts/release-tools` is a good place for this.